diff --git a/CMakeLists.txt b/CMakeLists.txt
index c6459b6..6281370 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,12 +18,6 @@ check_struct_has_member("struct msghdr" "msg_flags" "sys/types.h;sys/socket.h" H
 set(CMAKE_EXTRA_INCLUDE_FILES "sys/types.h" "sys/socket.h")
 check_type_size("socklen_t" HAS_SOCKLEN_T BUILTIN_TYPES_ONLY)
 unset(CMAKE_EXTRA_INCLUDE_FILES)
-if(MSVC)
-	add_definitions(-W3)
-else()
-	add_definitions(-Wno-error)
-endif()
-
 if(HAS_FCNTL)
     add_definitions(-DHAS_FCNTL=1)
 endif()
